draft-ietf-lamps-csr-attestation.md

```diff
@@ -86,7 +86,7 @@ for inclusion in PKCS#10 or CRMF certificate signing requests: an ASN.1 Attribut
 # Introduction
 
 At the time that it is requesting a certificate from a Certification Authority (CA), a PKI end entity may wish to provide evidence of the security properties of the environment in which the private key is stored.
-This evidence is to be verified by a relying party such as the Registration Authority or the Certification Authority as part of validating an incoming certificate request against a given certificate policy.
+This evidence is to be verified by a Verifier and the outcome of this evaluation is then made available to the Registration Authority (RA) or the Certification Authority (CA) as part of validating an incoming certificate request against a given certificate policy.
```

Comment:
```suggestion
This Evidence can be appraised by authoritative entities, such as a Registration Authority (RA) or a CA as part of validating an incoming certificate request against given certificate policies.
```

I'd prefer the phrasing that I proposed in #88, as we do not have to, for example, reintroduce `CA` that was just introduced in line 88 above and #88 also fixes inconsistent language, such as "evidence is to be verified".